examples/button.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
```

Comment:
maybe change file name from `button` to `neon-button` 
could your resolve the conflicts please.